src/simulation/ApplyLoad.h

```diff
@@ -72,6 +72,7 @@ class ApplyLoad
 
     LedgerKey mLoadCodeKey;
     TxGenerator::ContractInstance mLoadInstance;
+    TxGenerator::ContractInstance mXLMInstance;
```

Comment:
Nit: maybe clarify names a bit, like mSACInstance with a comment about the difference between what load these instances are expected to be used for.

Reply:
Done